pkg/agent/sysadvisor/plugin/qosaware/resource/cpu/advisor_helper.go

```diff
@@ -272,20 +277,60 @@ func (cra *cpuResourceAdvisor) updateRegionHeadroom() {
 	for regionName, r := range cra.regionMap {
 		regionInfo, ok := cra.metaCache.GetRegionInfo(regionName)
 		if !ok {
+			general.InfoS("failed to find region", "regionName", regionName)
 			continue
 		}
 
 		headroom, err := r.GetHeadroom()
 		if err != nil {
+			general.ErrorS(err, "failed to get region headroom", "regionName", r.Name())
 			continue
 		}
 		regionInfo.Headroom = headroom
 		regionInfo.HeadroomPolicyTopPriority, regionInfo.HeadroomPolicyInUse = r.GetHeadRoomPolicy()
 
+		general.InfoS("headroom policy", "HeadroomPolicyTopPriority", regionInfo.HeadroomPolicyTopPriority,
+			"HeadroomPolicyInUse", regionInfo.HeadroomPolicyInUse, "regionName", r.Name())
+
 		_ = cra.metaCache.SetRegionInfo(regionName, regionInfo)
 	}
 }
 
+func (cra *cpuResourceAdvisor) updateRegionEntries() {
+	entries := make(types.RegionEntries)
+	for regionName, r := range cra.regionMap {
+		regionInfo := &types.RegionInfo{
+			RegionName:    r.Name(),
+			RegionType:    r.Type(),
+			OwnerPoolName: r.OwnerPoolName(),
+			BindingNumas:  r.GetBindingNumas(),
+		}
+		headroom, err := r.GetHeadroom()
```

Comment:
Note: GetHeadroom may still be called more than one time during an update period, resulting in duplicate region policy update and metrics.